src/SeqCli/SeqCli.csproj

```diff
@@ -10,6 +10,9 @@
     <TreatSpecificWarningsAsErrors />
     <Platforms>x64</Platforms>
     <LangVersion>8</LangVersion>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>seqcli</ToolCommandName>
+    <PackageOutputPath>../../artifacts</PackageOutputPath>
```

Comment:
Since the `artifacts` path is determined in the build script, can we remove this and pass it to `dotnet pack` using `-o <path>` instead?

Reply:
I will update to match